src/style.css

```diff
@@ -0,0 +1,105 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  display: block;
+  margin: 8px;
+}
+
+.form {
+  display: flex;
+  justify-content: left;
+  margin: 20px auto 0;
+  width: 100%;
+}
+
+.bigform {
+  height: 70px;
+}
+
+.smallform {
+  width: 100%;
+  height: 42px;
+}
+
+.barcontainer {
+  width: 100%;
+  position: relative;
+  display: block;
+}
+
+.barcontainer--1::after {
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+  background-image: url(./images/Vector.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+}
+
+.barcontainer--2::after {
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+  background-image: url(./images/Icon.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+}
+
+.barcontainer::after {
```

Comment:
these properties are also the same for both, write them here:

background-repeat: no-repeat;
background-size: contain;

Reply:
fixed